app/src/main/java/com/android/sample/ui/profile/Profile.kt

```diff
@@ -313,8 +330,56 @@ fun ActivityRow(activity: Activity, onClickAction: () -> Unit, testTag: String)
               color = Color.Black)
           Text(text = activity.description, fontSize = SUBTITLE_FONTSIZE.sp, color = Color.Gray)
         }
+        if (category == "past") {
+          ReviewActivityButtons(activity.likes[userId]) { review ->
+            listActivitiesViewModel.reviewActivity(activity, userId, review)
+          }
+        }
       }
 }
+
+@Composable
+fun ReviewActivityButtons(currentReview: Boolean?, review: (Boolean?) -> Unit) {
+  var isLiked: Boolean? by remember { mutableStateOf(currentReview) }
+  Row {
+    IconButton(
+        onClick = {
+          isLiked = if (isLiked == true) null else true
+          review(isLiked)
+        },
+        colors =
+            IconButtonDefaults.iconButtonColors(
+                containerColor = if (isLiked == true) Color(0xFF048531) else Color.Transparent,
```

Comment:
Try to use a constant color instead of a hardcoded one

Reply:
Okay I will
Thanks.